services/notification/src/app.py

```diff
@@ -254,7 +254,31 @@ def null_to_none(s):
 
 
 def string_to_bool(s):
-    return None if s == 'null' else s == 'true'
+    """
+    Convert String to Bool
+
+    >>> string_to_bool('True')
+    True
+
+    >>> string_to_bool('true')
+    True
+
+    >>> string_to_bool('False')
+    False
+
+    >>> string_to_bool('false')
+    False
+
+    >>> string_to_bool('null')
```

Comment:
I don't like that this function returns None for 'null' but I'm leaving it since that's how it works. Also technically it returned False for None but I think the expectation given how everything else works is that it really should return None for None, so that's what I've done.